card/spec/lib/card/subcards_spec.rb

```diff
@@ -0,0 +1,78 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Subcards do
+
+  describe 'creating card with subcards' do
+    it 'works with subcard hash' do
+      Card::Auth.as_bot do
+        @card = Card.create! :name=>'card with subs', :subcards=> { '+sub1' => { :content => 'this is sub1'} }
+      end
+      expect(Card['card with subs+sub1'].content).to eq 'this is sub1'
+    end
+
+    it 'works with content string' do
+      Card::Auth.as_bot do
+        @card = Card.create! :name=>'card with subs', :subcards=> { '+sub1' => 'this is sub1'}
+      end
+      expect(Card['card with subs+sub1'].content).to eq 'this is sub1'
+    end
+
+    it 'works with +name key in args' do
+      Card::Auth.as_bot do
+        @card = Card.create! :name=>'card with subs', '+sub1' => { :content => 'this is sub1'}
+      end
+      expect(Card['card with subs+sub1'].content).to eq 'this is sub1'
+    end
+
+    it 'handles more than one level' do
+      Card::Auth.as_bot do
+        @card = Card.create! :name=>'card with subs', '+sub1' => { '+sub2' => 'this is sub2' }
+      end
+      expect(Card['card with subs+sub1+sub2'].content).to eq 'this is sub2'
+    end
+
+    it 'handles compound names' do
+      Card::Auth.as_bot do
+        @card = Card.create! :name=>'superman', '+*account+*email' => 'clark@sky.com'
```

Comment:
Line is too long. [85/80]<br>Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.